plotter/scatterColor_example_test.go

```diff
@@ -75,7 +74,8 @@ func ExampleScatter_color() {
 		_, _, z := scatterData.XYZ(i)
 		d := (z - minZ) / (maxZ - minZ)
 		rng := maxZ - minZ
-		k := d*rng + minZ
+		// Possible floating point error, similar to https://github.com/gonum/plot/issues/798,
+		k := min(d*rng+minZ, maxZ)
```

Comment:
indeed, it seems to possibly be prey to the same bug than #798.
I'll leave it to @kortschak to decide whether he'd prefer to deal with it in the same way.
(I'm fine with clamping the value like that)

Reply:
We don't need the comment, at least, we don't need a reference to the issue. There can be an explanatory not that this is required due to float error.

However, I'd put it in the call, `c, err := colors.At(min(k, colors.Max()))` and leave the original `k := d*rng + minZ`.

Reply:
This is not done; use `k := d*rng + minZ` rather than calling `min` twice.